Documentation/ProjectFile/prj/processes/process/RICHARDS_FLOW/material_property/porous_medium/porous_medium/a_id.md

```diff
@@ -0,0 +1,2 @@
+It specifies the material ID of a porous medium. The material IDs of elements
+should be given in the mesh's cell data array "MaterialIDs".
```

Comment:
The LiquidFlow, HT and ComponentTransport processes use a 'homogeneous' mode if the PropertyVector 'MaterialIDs' is not available in the mesh properties. In my opinion ogs should crash in this case. But this is/should not be part of this PR.